src/D2L.CodeStyle.Analyzers/Async/Generator/AsyncToSyncMethodTransformer.cs

```diff
@@ -458,6 +461,17 @@ private VariableDesignationSyntax Transform( VariableDesignationSyntax des )
 		);
 	}
 
+	private SyntaxList<InterpolatedStringContentSyntax> TransformInterpolations( SyntaxList<InterpolatedStringContentSyntax> interSyntaxs ) {
+		SyntaxList <InterpolatedStringContentSyntax> newInterSyntaxs = interSyntaxs;

```

Comment:
`newInterSyntaxs` isn't really adding anything here, right? `interSyntaxs = interSyntaxs.Replace( interSyntaxs[i]` would be equivalent?

Reply:
There was an issue where the first replace would change the syntax location of the later ones in interSyntaxs, meaning they could no longer be found in the SyntaxTree causing an exception on a type check somewhere else. Using newInterSyntaxs solves that